<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<!-- Begin Jekyll SEO tag v2.7.1 -->
<title>Image-derived phenotype extraction for genetic discovery via unsupervised deep learning in CMR images | MICCAI 2021 - Accepted Papers and Reviews</title>
<meta name="generator" content="Jekyll v3.9.0" />
<meta property="og:title" content="Image-derived phenotype extraction for genetic discovery via unsupervised deep learning in CMR images" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Paper Info Reviews Meta-review(s) Author Feedback Authors Rodrigo Bonazzola, Nishant Ravikumar, Rahman Attar, Enzo Ferrante, Tanveer Syeda-Mahmood, Alejandro F. Frangi Abstract Prospective studies with linked image and genetic data, suchas the UK Biobank (UKB), provide an unprecedented opportunity to extract knowledge on the genetic basis of image-derived phenotypes. However, the extent of phenotypes tested within so-called genome-wide association studies (GWAS) is usually limited to handcrafted features, where the main limitation to proceed otherwise is the high dimensionality of both the imaging and genetic data. Here, we propose an approach where the phenotyping is performed in an unsupervised manner, via autoencoders that operate on image-derived 3D meshes. Therefore, the latent variables produced by the encoder condense the information related to the geometry of the biologic structure of interest. The network’s training proceeds in two steps: the first is genotype-agnostic and the second enforces an association with a set of genetic markers selected via GWAS on the intermediate latent representation. This genotype-dependent optimisation procedure allows the refinement of the phenotypes produced by the autoencoder to better understand the effect of the genetic markers encountered. We tested and validated our proposed method on left-ventricular meshes derived from cardiovascular magnetic resonance images from the UKB, leading to the discovery of novel genetic associations that, to the best of our knowledge, had not been yet reported in the literature on cardiac phenotypes. Link to paper https://doi.org/10.1007/978-3-030-87240-3_67 Link to the code repository N/A Link to the dataset(s) N/A Reviews Review #1 Please describe the contribution of the paper This paper proposes an unsupervised deep learning method to study the genetic basis of CMR-derived phenotypes. The method contains two-step. The first step is to extract intermediate representations of imaging data, and the second step is to study the association between this intermediate variable and the genetic data. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. This paper proposed an unsupervised learning method to extract the respretations of the imaging data, and then used the phenotype data to conduct a GWAS to analyze the association between imaging and genetics. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The motivation is not clearly presented. It looks hard to interpret the extracted imaging feature. There were many grammatical mistakes. Please rate the clarity and organization of this paper Poor Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Fair. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html Please clearly present the motivation of this work. Do you want to identify new risk loci? Or do you want to extract a good intermediate imaging features? As an deep learning method, how about its performance improvement compared to conventional GWAS? Please state your overall opinion of the paper strong reject (2) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? Confident. What is the ranking of this paper in your review stack? 8 Number of papers in your stack 3 Reviewer confidence Very confident Review #2 Please describe the contribution of the paper The manuscript proposed an innovative autoencoder model based on graph convolutional neural networks and optimized the model using a collaborative approach. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. It is very innovative approach to integrate imaging and genetics using autoencoder with collaborative learning, as presented in Fig 1. The author well described the motivation and contibution of the proposed model. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. It would be better if the author presents the rich comparisons with several baseline algorithm, such as CCA or PCA. Please rate the clarity and organization of this paper Excellent Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance The paper clearly described their implementations. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html The paper is overall nicely written with interesting ideas. However, there are several questions I had that could help me better assess the value of the proposed methods, especially in the experiment. There’s no need for running additional experiments for any of my questions. Please state your overall opinion of the paper strong accept (9) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? The proposed model is very innovative. In the manuscript, the author proposed a novel cost function to integrate imaging and genetics. The motivation of the cost function was very concrete. What is the ranking of this paper in your review stack? 1 Number of papers in your stack 3 Reviewer confidence Very confident Review #3 Please describe the contribution of the paper The paper presents an auto-encoder of 3D surface mesh models of the heart’s left ventricle that is simultaneously optimized to improve the odds of a significant correlation in a Genome-Wide Association Study. The authors train the encoder in two stages: first, by using graph convolutions on the mesh based on the graph Laplacian, and second by adding a term expressing preference for encoding components that have greater correlation with a selected set of SNPs. The authors train, validate, and test the pipeline on 29000 subjects from the UK Biobank, claiming to discover a new heart-related SNP. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. The paper addresses an important area in imaging genetics, i.e. maximizing discoverability. GWAS is a bit old-hat, but remains a useful screening tool for yet-undiscovered variants, knowledge of which can be passed on to basic science. The choice of spectral embedding with a Chebyshev polynomial kernel is reasonable, and the pre-processing of the data appears to be done thoroughly and well. The learning &amp; GWAS study is done on a sufficiently large sample, and the validation of the method is solid, with varying seeds for splitting the data and assessing the p-value distribution for a each of a set of hyperparameters. That the authors connect their putative discovery with an interpretable trait (sphericity) by way of the auto-encoder component is also a bonus. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The novelty of the method is somewhat limited, although this can be forgiven, given the goal of improving GWAS discovery. The main weakness of the method is that it relies on optimizing the encoder components’ correlation with SNP’s that are already sufficiently strongly correlated to pass Bonferroni correction. This type of approach appears circular in its reasoning. Although not circular in the statistical/ML sense, this is still a recipe for spurious findings. To properly optimize complex traits for “discoverability,” the process has to be agnostic to specific variants, especially from the very same dataset. More sensible approaches target general heritability, defined for example by GCTA https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363/. Even approaches like this https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5946904/ use overall GWAS from external data to define “more discoverable” traits. The most likely outcome here is that the same variant would be putatively discovered without the second (genetic) stage of the auto-encoder training. As a more minor point, the autoencoder is comprised of only 4 components in the end (after stage 1), which makes the entire procedure seem somewhat unnecessary, given the low dimensionality to begin with. Also, it seems the initial auto-encoding was done on the entire dataset at once (although this is not totally clear from the algorithm outline), which would obviously bias the results further. Please rate the clarity and organization of this paper Very Good Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Authors include code in the supplement &amp; describe the tools for pre-processing. Data is UKBB, so can be accessed with some effort by any researcher. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html See weaknesses Please state your overall opinion of the paper borderline reject (5) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? At the end, the paper addresses an important issue, but the simplistic approach to the genomics aspect makes this a work of limited value. What is the ranking of this paper in your review stack? 3 Number of papers in your stack 3 Reviewer confidence Very confident Meta-Review(s) Primary Meta-Review Please provide your assessment of this work, taking into account all reviews. Summarize the key strengths and weaknesses of the paper and justify your recommendation. In case you deviate from the reviewers’ recommendations, explain in detail the reasons why. In case of an invitation for rebuttal, clarify which points are important to address in the rebuttal. The reviewers recognize the interest of the presented approach to enhance discoverability of relevant imaging-genetics associations. The presented autoencoding framework was found sound while the experimental assessment on the UK Biobank is compelling. The paper seems however to suffer from lack of clarity in illustrating the benefits from the proposed two steps approach over the conventional GWAS analysis. There seems to be an issue with the circularity of the analysis, as both screening and validation of variants are performed on the same dataset. What is the ranking of this paper in your stack? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 5 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The relevance of the approach is discussed, in particular with respect to the optimization results shown in Figure 3. Concerning the circularity issue, it is made clear that the multi-step approach is performed on disjoint discovery and validation sets. Overall, the rebuttal is satisfactory in addressing the major concerns expressed during the review. Due to the originality of the proposed approach, and the large sample considered in the study, the feeling is that the work could make a nice contribution to the conference. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 7 Meta-Review #2 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The authors have addressed the clarity with respect to the key circularity issue. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 6 Meta-Review #3 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. This paper proposes to learn genetic associations with image-based phenotypes using unsupervised graph auto encoder. Validation is on left ventricular shapes from the uk biobank. One reviewer finds the motivation unclear. A second reviewer finds it clear but finds it lacking better comparisons. A third reviewer appreciates the method but questions whether discoveries may be due to circular reasoning. Reviews are extremely disagreeing, however, two of which are too short, and in my opinion, fail to meet expectations from a reviewer. The most constructive review brings a point on circularity. In other words, optimizing the encoder components with genetic snp’s may lead to spurious discoveries. From my understanding of the paper and from the author’s rebuttal, the approach is not to necessarily discover new snp’s but rather provided richer, fine-tuning of phenotyping shape descriptors. The algorithmic contribution uses existing methods, less original in that aspect, but their use to fine-tune phenotyping with an extensive validation is, in my opinion, a contribution with potential impact worth publishing. For these reasons, Recommendation is toward Acceptance. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 11 Author Feedback We thank all the reviewers for their comments. Overall, they seem to concur on the importance of this emerging field and our general approach/novelty. The main comments to be addressed concern circularity and comparison to older statistical methods. To clarify for Reviewer-1, our method in the training consists of 3 (not two as stated in the review) steps: the first step extracts intermediate representations of imaging data, the second step studies the association between the shape features and the genetic data using GWAS, the third step incorporates the salient SNP found back into the estimation of shape features through a new loss function. Finally, during inference, we show the improvement in the strength of the association of the shape features with the genomic loci from the updated shape features. Reviewer-2: Although it was not clear from the comment, we assume the comparison reference to PCA was for the shape representation and CCA was for the purpose of GWAS association. Our prior work using statistical methods such as PCA instead of the graph convolutional mesh revealed no meaningful GWAS associations. Since the mesh encoder captures the relationship between voxels inherent in the finite element mesh modeling, the resulting embedding is more suitable for finding the associations in GWAS. Our focus was on discovering relevant shape features associated with genetic loci without pre-programming GWAS with selected features as done in conventional GWAS. However, it is possible that CCA will be useful for GWAS itself although sparse sample graph-based CCA on such large dimensions will be computationally expensive requiring matrix deflation techniques. Reviewer-3: We emphasize that, to our knowledge, our paper is the first work to extract deep shape features for genetic discovery using unsupervised geometric deep learning. It makes two specific novel contributions: 1) use of variationally regularized convolutional mesh autoencoders on graphs for the purpose of discovery of genetic associations of shape features. 2) using a genome-dependent loss function to optimize the shape representation for genetic association. Previous work on 3D facial meshes relied on more traditional ML approaches like hierarchical clustering and PCA on shape for GWAS association. On the circularity question, we clarify that the purpose of the second training step is to find a refined representation (i.e. a better phenotype) for association with genetics, which stresses the morphological changes that led to increase such association. It was not intended to discover newer gene loci. We found that a term enforcing correlation of a latent variable with SNP dosage is indeed effective in improving the association in an independent sample. This is not obvious a priori due to the small effect of single genetic variants on the phenotype. Finally, there is no circularity in the statistical/ML sense, since no optimization is performed on the GWAS sample. We can assume a uniform distribution of p-values under the null distribution of no effect, which in turn allows us to perform hypothesis testing in the usual way without increasing the risk of spurious findings. While the improvement in association is modest, it is still statistically significant as shown in Fig. 3, with w_{SNP}=0 corresponding to no optimization; this supports its inclusion in the paper. We believe that by taking special measures during training, as well as using a larger sample size, the optimization can be further improved. We further argue that in a more general scenario, this refinement could also help find more true-positive associations for the case of a polygenic trait. We also clarify that the testing of the improved association (i.e. in final GWAS) was performed on a disjoint patient dataset from the one used for initial encoding, thus avoiding the bias due to overlap of datasets. The paper will be thoroughly revised for any grammatical errors in the final version. back to top" />
<meta property="og:description" content="Paper Info Reviews Meta-review(s) Author Feedback Authors Rodrigo Bonazzola, Nishant Ravikumar, Rahman Attar, Enzo Ferrante, Tanveer Syeda-Mahmood, Alejandro F. Frangi Abstract Prospective studies with linked image and genetic data, suchas the UK Biobank (UKB), provide an unprecedented opportunity to extract knowledge on the genetic basis of image-derived phenotypes. However, the extent of phenotypes tested within so-called genome-wide association studies (GWAS) is usually limited to handcrafted features, where the main limitation to proceed otherwise is the high dimensionality of both the imaging and genetic data. Here, we propose an approach where the phenotyping is performed in an unsupervised manner, via autoencoders that operate on image-derived 3D meshes. Therefore, the latent variables produced by the encoder condense the information related to the geometry of the biologic structure of interest. The network’s training proceeds in two steps: the first is genotype-agnostic and the second enforces an association with a set of genetic markers selected via GWAS on the intermediate latent representation. This genotype-dependent optimisation procedure allows the refinement of the phenotypes produced by the autoencoder to better understand the effect of the genetic markers encountered. We tested and validated our proposed method on left-ventricular meshes derived from cardiovascular magnetic resonance images from the UKB, leading to the discovery of novel genetic associations that, to the best of our knowledge, had not been yet reported in the literature on cardiac phenotypes. Link to paper https://doi.org/10.1007/978-3-030-87240-3_67 Link to the code repository N/A Link to the dataset(s) N/A Reviews Review #1 Please describe the contribution of the paper This paper proposes an unsupervised deep learning method to study the genetic basis of CMR-derived phenotypes. The method contains two-step. The first step is to extract intermediate representations of imaging data, and the second step is to study the association between this intermediate variable and the genetic data. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. This paper proposed an unsupervised learning method to extract the respretations of the imaging data, and then used the phenotype data to conduct a GWAS to analyze the association between imaging and genetics. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The motivation is not clearly presented. It looks hard to interpret the extracted imaging feature. There were many grammatical mistakes. Please rate the clarity and organization of this paper Poor Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Fair. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html Please clearly present the motivation of this work. Do you want to identify new risk loci? Or do you want to extract a good intermediate imaging features? As an deep learning method, how about its performance improvement compared to conventional GWAS? Please state your overall opinion of the paper strong reject (2) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? Confident. What is the ranking of this paper in your review stack? 8 Number of papers in your stack 3 Reviewer confidence Very confident Review #2 Please describe the contribution of the paper The manuscript proposed an innovative autoencoder model based on graph convolutional neural networks and optimized the model using a collaborative approach. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. It is very innovative approach to integrate imaging and genetics using autoencoder with collaborative learning, as presented in Fig 1. The author well described the motivation and contibution of the proposed model. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. It would be better if the author presents the rich comparisons with several baseline algorithm, such as CCA or PCA. Please rate the clarity and organization of this paper Excellent Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance The paper clearly described their implementations. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html The paper is overall nicely written with interesting ideas. However, there are several questions I had that could help me better assess the value of the proposed methods, especially in the experiment. There’s no need for running additional experiments for any of my questions. Please state your overall opinion of the paper strong accept (9) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? The proposed model is very innovative. In the manuscript, the author proposed a novel cost function to integrate imaging and genetics. The motivation of the cost function was very concrete. What is the ranking of this paper in your review stack? 1 Number of papers in your stack 3 Reviewer confidence Very confident Review #3 Please describe the contribution of the paper The paper presents an auto-encoder of 3D surface mesh models of the heart’s left ventricle that is simultaneously optimized to improve the odds of a significant correlation in a Genome-Wide Association Study. The authors train the encoder in two stages: first, by using graph convolutions on the mesh based on the graph Laplacian, and second by adding a term expressing preference for encoding components that have greater correlation with a selected set of SNPs. The authors train, validate, and test the pipeline on 29000 subjects from the UK Biobank, claiming to discover a new heart-related SNP. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. The paper addresses an important area in imaging genetics, i.e. maximizing discoverability. GWAS is a bit old-hat, but remains a useful screening tool for yet-undiscovered variants, knowledge of which can be passed on to basic science. The choice of spectral embedding with a Chebyshev polynomial kernel is reasonable, and the pre-processing of the data appears to be done thoroughly and well. The learning &amp; GWAS study is done on a sufficiently large sample, and the validation of the method is solid, with varying seeds for splitting the data and assessing the p-value distribution for a each of a set of hyperparameters. That the authors connect their putative discovery with an interpretable trait (sphericity) by way of the auto-encoder component is also a bonus. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The novelty of the method is somewhat limited, although this can be forgiven, given the goal of improving GWAS discovery. The main weakness of the method is that it relies on optimizing the encoder components’ correlation with SNP’s that are already sufficiently strongly correlated to pass Bonferroni correction. This type of approach appears circular in its reasoning. Although not circular in the statistical/ML sense, this is still a recipe for spurious findings. To properly optimize complex traits for “discoverability,” the process has to be agnostic to specific variants, especially from the very same dataset. More sensible approaches target general heritability, defined for example by GCTA https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363/. Even approaches like this https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5946904/ use overall GWAS from external data to define “more discoverable” traits. The most likely outcome here is that the same variant would be putatively discovered without the second (genetic) stage of the auto-encoder training. As a more minor point, the autoencoder is comprised of only 4 components in the end (after stage 1), which makes the entire procedure seem somewhat unnecessary, given the low dimensionality to begin with. Also, it seems the initial auto-encoding was done on the entire dataset at once (although this is not totally clear from the algorithm outline), which would obviously bias the results further. Please rate the clarity and organization of this paper Very Good Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Authors include code in the supplement &amp; describe the tools for pre-processing. Data is UKBB, so can be accessed with some effort by any researcher. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html See weaknesses Please state your overall opinion of the paper borderline reject (5) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? At the end, the paper addresses an important issue, but the simplistic approach to the genomics aspect makes this a work of limited value. What is the ranking of this paper in your review stack? 3 Number of papers in your stack 3 Reviewer confidence Very confident Meta-Review(s) Primary Meta-Review Please provide your assessment of this work, taking into account all reviews. Summarize the key strengths and weaknesses of the paper and justify your recommendation. In case you deviate from the reviewers’ recommendations, explain in detail the reasons why. In case of an invitation for rebuttal, clarify which points are important to address in the rebuttal. The reviewers recognize the interest of the presented approach to enhance discoverability of relevant imaging-genetics associations. The presented autoencoding framework was found sound while the experimental assessment on the UK Biobank is compelling. The paper seems however to suffer from lack of clarity in illustrating the benefits from the proposed two steps approach over the conventional GWAS analysis. There seems to be an issue with the circularity of the analysis, as both screening and validation of variants are performed on the same dataset. What is the ranking of this paper in your stack? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 5 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The relevance of the approach is discussed, in particular with respect to the optimization results shown in Figure 3. Concerning the circularity issue, it is made clear that the multi-step approach is performed on disjoint discovery and validation sets. Overall, the rebuttal is satisfactory in addressing the major concerns expressed during the review. Due to the originality of the proposed approach, and the large sample considered in the study, the feeling is that the work could make a nice contribution to the conference. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 7 Meta-Review #2 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The authors have addressed the clarity with respect to the key circularity issue. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 6 Meta-Review #3 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. This paper proposes to learn genetic associations with image-based phenotypes using unsupervised graph auto encoder. Validation is on left ventricular shapes from the uk biobank. One reviewer finds the motivation unclear. A second reviewer finds it clear but finds it lacking better comparisons. A third reviewer appreciates the method but questions whether discoveries may be due to circular reasoning. Reviews are extremely disagreeing, however, two of which are too short, and in my opinion, fail to meet expectations from a reviewer. The most constructive review brings a point on circularity. In other words, optimizing the encoder components with genetic snp’s may lead to spurious discoveries. From my understanding of the paper and from the author’s rebuttal, the approach is not to necessarily discover new snp’s but rather provided richer, fine-tuning of phenotyping shape descriptors. The algorithmic contribution uses existing methods, less original in that aspect, but their use to fine-tune phenotyping with an extensive validation is, in my opinion, a contribution with potential impact worth publishing. For these reasons, Recommendation is toward Acceptance. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 11 Author Feedback We thank all the reviewers for their comments. Overall, they seem to concur on the importance of this emerging field and our general approach/novelty. The main comments to be addressed concern circularity and comparison to older statistical methods. To clarify for Reviewer-1, our method in the training consists of 3 (not two as stated in the review) steps: the first step extracts intermediate representations of imaging data, the second step studies the association between the shape features and the genetic data using GWAS, the third step incorporates the salient SNP found back into the estimation of shape features through a new loss function. Finally, during inference, we show the improvement in the strength of the association of the shape features with the genomic loci from the updated shape features. Reviewer-2: Although it was not clear from the comment, we assume the comparison reference to PCA was for the shape representation and CCA was for the purpose of GWAS association. Our prior work using statistical methods such as PCA instead of the graph convolutional mesh revealed no meaningful GWAS associations. Since the mesh encoder captures the relationship between voxels inherent in the finite element mesh modeling, the resulting embedding is more suitable for finding the associations in GWAS. Our focus was on discovering relevant shape features associated with genetic loci without pre-programming GWAS with selected features as done in conventional GWAS. However, it is possible that CCA will be useful for GWAS itself although sparse sample graph-based CCA on such large dimensions will be computationally expensive requiring matrix deflation techniques. Reviewer-3: We emphasize that, to our knowledge, our paper is the first work to extract deep shape features for genetic discovery using unsupervised geometric deep learning. It makes two specific novel contributions: 1) use of variationally regularized convolutional mesh autoencoders on graphs for the purpose of discovery of genetic associations of shape features. 2) using a genome-dependent loss function to optimize the shape representation for genetic association. Previous work on 3D facial meshes relied on more traditional ML approaches like hierarchical clustering and PCA on shape for GWAS association. On the circularity question, we clarify that the purpose of the second training step is to find a refined representation (i.e. a better phenotype) for association with genetics, which stresses the morphological changes that led to increase such association. It was not intended to discover newer gene loci. We found that a term enforcing correlation of a latent variable with SNP dosage is indeed effective in improving the association in an independent sample. This is not obvious a priori due to the small effect of single genetic variants on the phenotype. Finally, there is no circularity in the statistical/ML sense, since no optimization is performed on the GWAS sample. We can assume a uniform distribution of p-values under the null distribution of no effect, which in turn allows us to perform hypothesis testing in the usual way without increasing the risk of spurious findings. While the improvement in association is modest, it is still statistically significant as shown in Fig. 3, with w_{SNP}=0 corresponding to no optimization; this supports its inclusion in the paper. We believe that by taking special measures during training, as well as using a larger sample size, the optimization can be further improved. We further argue that in a more general scenario, this refinement could also help find more true-positive associations for the case of a polygenic trait. We also clarify that the testing of the improved association (i.e. in final GWAS) was performed on a disjoint patient dataset from the one used for initial encoding, thus avoiding the bias due to overlap of datasets. The paper will be thoroughly revised for any grammatical errors in the final version. back to top" />
<link rel="canonical" href="https://kittywong.github.io/kittywong/0566/12/31/Paper0968" />
<meta property="og:url" content="https://kittywong.github.io/kittywong/0566/12/31/Paper0968" />
<meta property="og:site_name" content="MICCAI 2021 - Accepted Papers and Reviews" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="0566-12-31T23:58:56-05:17" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Image-derived phenotype extraction for genetic discovery via unsupervised deep learning in CMR images" />
<script type="application/ld+json">
{"mainEntityOfPage":{"@type":"WebPage","@id":"https://kittywong.github.io/kittywong/0566/12/31/Paper0968"},"@type":"BlogPosting","url":"https://kittywong.github.io/kittywong/0566/12/31/Paper0968","headline":"Image-derived phenotype extraction for genetic discovery via unsupervised deep learning in CMR images","dateModified":"0567-01-03T00:00:00-05:17","datePublished":"0566-12-31T23:58:56-05:17","description":"Paper Info Reviews Meta-review(s) Author Feedback Authors Rodrigo Bonazzola, Nishant Ravikumar, Rahman Attar, Enzo Ferrante, Tanveer Syeda-Mahmood, Alejandro F. Frangi Abstract Prospective studies with linked image and genetic data, suchas the UK Biobank (UKB), provide an unprecedented opportunity to extract knowledge on the genetic basis of image-derived phenotypes. However, the extent of phenotypes tested within so-called genome-wide association studies (GWAS) is usually limited to handcrafted features, where the main limitation to proceed otherwise is the high dimensionality of both the imaging and genetic data. Here, we propose an approach where the phenotyping is performed in an unsupervised manner, via autoencoders that operate on image-derived 3D meshes. Therefore, the latent variables produced by the encoder condense the information related to the geometry of the biologic structure of interest. The network’s training proceeds in two steps: the first is genotype-agnostic and the second enforces an association with a set of genetic markers selected via GWAS on the intermediate latent representation. This genotype-dependent optimisation procedure allows the refinement of the phenotypes produced by the autoencoder to better understand the effect of the genetic markers encountered. We tested and validated our proposed method on left-ventricular meshes derived from cardiovascular magnetic resonance images from the UKB, leading to the discovery of novel genetic associations that, to the best of our knowledge, had not been yet reported in the literature on cardiac phenotypes. Link to paper https://doi.org/10.1007/978-3-030-87240-3_67 Link to the code repository N/A Link to the dataset(s) N/A Reviews Review #1 Please describe the contribution of the paper This paper proposes an unsupervised deep learning method to study the genetic basis of CMR-derived phenotypes. The method contains two-step. The first step is to extract intermediate representations of imaging data, and the second step is to study the association between this intermediate variable and the genetic data. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. This paper proposed an unsupervised learning method to extract the respretations of the imaging data, and then used the phenotype data to conduct a GWAS to analyze the association between imaging and genetics. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The motivation is not clearly presented. It looks hard to interpret the extracted imaging feature. There were many grammatical mistakes. Please rate the clarity and organization of this paper Poor Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Fair. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html Please clearly present the motivation of this work. Do you want to identify new risk loci? Or do you want to extract a good intermediate imaging features? As an deep learning method, how about its performance improvement compared to conventional GWAS? Please state your overall opinion of the paper strong reject (2) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? Confident. What is the ranking of this paper in your review stack? 8 Number of papers in your stack 3 Reviewer confidence Very confident Review #2 Please describe the contribution of the paper The manuscript proposed an innovative autoencoder model based on graph convolutional neural networks and optimized the model using a collaborative approach. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. It is very innovative approach to integrate imaging and genetics using autoencoder with collaborative learning, as presented in Fig 1. The author well described the motivation and contibution of the proposed model. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. It would be better if the author presents the rich comparisons with several baseline algorithm, such as CCA or PCA. Please rate the clarity and organization of this paper Excellent Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance The paper clearly described their implementations. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html The paper is overall nicely written with interesting ideas. However, there are several questions I had that could help me better assess the value of the proposed methods, especially in the experiment. There’s no need for running additional experiments for any of my questions. Please state your overall opinion of the paper strong accept (9) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? The proposed model is very innovative. In the manuscript, the author proposed a novel cost function to integrate imaging and genetics. The motivation of the cost function was very concrete. What is the ranking of this paper in your review stack? 1 Number of papers in your stack 3 Reviewer confidence Very confident Review #3 Please describe the contribution of the paper The paper presents an auto-encoder of 3D surface mesh models of the heart’s left ventricle that is simultaneously optimized to improve the odds of a significant correlation in a Genome-Wide Association Study. The authors train the encoder in two stages: first, by using graph convolutions on the mesh based on the graph Laplacian, and second by adding a term expressing preference for encoding components that have greater correlation with a selected set of SNPs. The authors train, validate, and test the pipeline on 29000 subjects from the UK Biobank, claiming to discover a new heart-related SNP. Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting. The paper addresses an important area in imaging genetics, i.e. maximizing discoverability. GWAS is a bit old-hat, but remains a useful screening tool for yet-undiscovered variants, knowledge of which can be passed on to basic science. The choice of spectral embedding with a Chebyshev polynomial kernel is reasonable, and the pre-processing of the data appears to be done thoroughly and well. The learning &amp; GWAS study is done on a sufficiently large sample, and the validation of the method is solid, with varying seeds for splitting the data and assessing the p-value distribution for a each of a set of hyperparameters. That the authors connect their putative discovery with an interpretable trait (sphericity) by way of the auto-encoder component is also a bonus. Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work. The novelty of the method is somewhat limited, although this can be forgiven, given the goal of improving GWAS discovery. The main weakness of the method is that it relies on optimizing the encoder components’ correlation with SNP’s that are already sufficiently strongly correlated to pass Bonferroni correction. This type of approach appears circular in its reasoning. Although not circular in the statistical/ML sense, this is still a recipe for spurious findings. To properly optimize complex traits for “discoverability,” the process has to be agnostic to specific variants, especially from the very same dataset. More sensible approaches target general heritability, defined for example by GCTA https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363/. Even approaches like this https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5946904/ use overall GWAS from external data to define “more discoverable” traits. The most likely outcome here is that the same variant would be putatively discovered without the second (genetic) stage of the auto-encoder training. As a more minor point, the autoencoder is comprised of only 4 components in the end (after stage 1), which makes the entire procedure seem somewhat unnecessary, given the low dimensionality to begin with. Also, it seems the initial auto-encoding was done on the entire dataset at once (although this is not totally clear from the algorithm outline), which would obviously bias the results further. Please rate the clarity and organization of this paper Very Good Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance Authors include code in the supplement &amp; describe the tools for pre-processing. Data is UKBB, so can be accessed with some effort by any researcher. Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: https://miccai2021.org/en/REVIEWER-GUIDELINES.html See weaknesses Please state your overall opinion of the paper borderline reject (5) Please justify your recommendation. What were the major factors that led you to your overall score for this paper? At the end, the paper addresses an important issue, but the simplistic approach to the genomics aspect makes this a work of limited value. What is the ranking of this paper in your review stack? 3 Number of papers in your stack 3 Reviewer confidence Very confident Meta-Review(s) Primary Meta-Review Please provide your assessment of this work, taking into account all reviews. Summarize the key strengths and weaknesses of the paper and justify your recommendation. In case you deviate from the reviewers’ recommendations, explain in detail the reasons why. In case of an invitation for rebuttal, clarify which points are important to address in the rebuttal. The reviewers recognize the interest of the presented approach to enhance discoverability of relevant imaging-genetics associations. The presented autoencoding framework was found sound while the experimental assessment on the UK Biobank is compelling. The paper seems however to suffer from lack of clarity in illustrating the benefits from the proposed two steps approach over the conventional GWAS analysis. There seems to be an issue with the circularity of the analysis, as both screening and validation of variants are performed on the same dataset. What is the ranking of this paper in your stack? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 5 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The relevance of the approach is discussed, in particular with respect to the optimization results shown in Figure 3. Concerning the circularity issue, it is made clear that the multi-step approach is performed on disjoint discovery and validation sets. Overall, the rebuttal is satisfactory in addressing the major concerns expressed during the review. Due to the originality of the proposed approach, and the large sample considered in the study, the feeling is that the work could make a nice contribution to the conference. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 7 Meta-Review #2 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. The authors have addressed the clarity with respect to the key circularity issue. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 6 Meta-Review #3 Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores, indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision. This paper proposes to learn genetic associations with image-based phenotypes using unsupervised graph auto encoder. Validation is on left ventricular shapes from the uk biobank. One reviewer finds the motivation unclear. A second reviewer finds it clear but finds it lacking better comparisons. A third reviewer appreciates the method but questions whether discoveries may be due to circular reasoning. Reviews are extremely disagreeing, however, two of which are too short, and in my opinion, fail to meet expectations from a reviewer. The most constructive review brings a point on circularity. In other words, optimizing the encoder components with genetic snp’s may lead to spurious discoveries. From my understanding of the paper and from the author’s rebuttal, the approach is not to necessarily discover new snp’s but rather provided richer, fine-tuning of phenotyping shape descriptors. The algorithmic contribution uses existing methods, less original in that aspect, but their use to fine-tune phenotyping with an extensive validation is, in my opinion, a contribution with potential impact worth publishing. For these reasons, Recommendation is toward Acceptance. After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal. Accept What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers). 11 Author Feedback We thank all the reviewers for their comments. Overall, they seem to concur on the importance of this emerging field and our general approach/novelty. The main comments to be addressed concern circularity and comparison to older statistical methods. To clarify for Reviewer-1, our method in the training consists of 3 (not two as stated in the review) steps: the first step extracts intermediate representations of imaging data, the second step studies the association between the shape features and the genetic data using GWAS, the third step incorporates the salient SNP found back into the estimation of shape features through a new loss function. Finally, during inference, we show the improvement in the strength of the association of the shape features with the genomic loci from the updated shape features. Reviewer-2: Although it was not clear from the comment, we assume the comparison reference to PCA was for the shape representation and CCA was for the purpose of GWAS association. Our prior work using statistical methods such as PCA instead of the graph convolutional mesh revealed no meaningful GWAS associations. Since the mesh encoder captures the relationship between voxels inherent in the finite element mesh modeling, the resulting embedding is more suitable for finding the associations in GWAS. Our focus was on discovering relevant shape features associated with genetic loci without pre-programming GWAS with selected features as done in conventional GWAS. However, it is possible that CCA will be useful for GWAS itself although sparse sample graph-based CCA on such large dimensions will be computationally expensive requiring matrix deflation techniques. Reviewer-3: We emphasize that, to our knowledge, our paper is the first work to extract deep shape features for genetic discovery using unsupervised geometric deep learning. It makes two specific novel contributions: 1) use of variationally regularized convolutional mesh autoencoders on graphs for the purpose of discovery of genetic associations of shape features. 2) using a genome-dependent loss function to optimize the shape representation for genetic association. Previous work on 3D facial meshes relied on more traditional ML approaches like hierarchical clustering and PCA on shape for GWAS association. On the circularity question, we clarify that the purpose of the second training step is to find a refined representation (i.e. a better phenotype) for association with genetics, which stresses the morphological changes that led to increase such association. It was not intended to discover newer gene loci. We found that a term enforcing correlation of a latent variable with SNP dosage is indeed effective in improving the association in an independent sample. This is not obvious a priori due to the small effect of single genetic variants on the phenotype. Finally, there is no circularity in the statistical/ML sense, since no optimization is performed on the GWAS sample. We can assume a uniform distribution of p-values under the null distribution of no effect, which in turn allows us to perform hypothesis testing in the usual way without increasing the risk of spurious findings. While the improvement in association is modest, it is still statistically significant as shown in Fig. 3, with w_{SNP}=0 corresponding to no optimization; this supports its inclusion in the paper. We believe that by taking special measures during training, as well as using a larger sample size, the optimization can be further improved. We further argue that in a more general scenario, this refinement could also help find more true-positive associations for the case of a polygenic trait. We also clarify that the testing of the improved association (i.e. in final GWAS) was performed on a disjoint patient dataset from the one used for initial encoding, thus avoiding the bias due to overlap of datasets. The paper will be thoroughly revised for any grammatical errors in the final version. back to top","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


<meta
  name="keywords"
  content="Bonazzola, Rodrigo,Ravikumar, Nishant,Attar, Rahman,Ferrante, Enzo,Syeda-Mahmood, Tanveer,Frangi, Alejandro F." />

<link rel="shortcut icon" href="/favicon.ico" />
<link rel="apple-touch-icon" href="/favicon.ico" />
<link
  rel="alternate"
  type="application/rss+xml"
  title="MICCAI 2021 - Accepted Papers and Reviews - "
  href="kittywong/feed.xml" />
<link
  rel="stylesheet"
  type="text/css"
  href="kittywong/assets/css/base.css" />
<link
  rel="stylesheet"
  type="text/css"
  href="kittywong/assets/css/highlight.css" />

<!--[if lt IE 9]>
  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->


<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  async>
</script>


<script src="/assets/scripts/jekyllpaper.js" async></script>
<script
  src="https://unpkg.com/mermaid@8.5.1/dist/mermaid.min.js"
  onload="javascript:loadMermaid();"
  async>
</script>

<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    jax: ["input/TeX", "output/HTML-CSS"],
    tex2jax: {
      inlineMath: [ ['$', '$'], ["\\(", "\\)"] ],
      displayMath: [ ['$$', '$$'], ["\\[", "\\]"] ],
      processEscapes: true,
      skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
    }
    //,
    //displayAlign: "left",
    //displayIndent: "2em"
  });
</script>
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML" type="text/javascript"></script>

  </head>
  <body>
    <div class="container-wrapper">
      <header class="container-header">
        <div class="header-info">
  <!-- Added by Elvis Chen -->
  <img src="https://kittywong.github.io/assets/images/miccai2021header.jpg" alt="MICCAI banner">
  <!-- END Added by Elvis Chen -->
  <span class="header-info-name">MICCAI 2021 - Accepted Papers and Reviews</span>
  <span class="header-info-desc"></span>
</div>
<nav class="header-nav">
  <ul class="header-main-nav">
    <h2>
    
    <li class="header-main-nav-item">
      <a href="kittywong/">
        
          <b>List of Papers</b>
        
      </a>
    </li>
      
    <li class="header-main-nav-item">
      <a href="kittywong/categories">
        
          <b>By topics</b>
        
      </a>
    </li>
      
    <li class="header-main-nav-item">
      <a href="kittywong/tags">
        
          <b>Author List</b>
        
      </a>
    </li>
      
    <li class="header-main-nav-item">
      <a href="kittywong/about">
        
          <b>About</b>
        
      </a>
    </li>
      
    </h2>
  </ul>
</nav>
      </header>
      <main class="container-main">
        <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>

<article class="container-post">
  <div class="post-title">
    <h1><span style="color:#1040a7"><b>Image-derived phenotype extraction for genetic discovery via unsupervised deep learning in CMR images</b></span></h1>
  </div>
  
  <div class="post-author print-post-author">
    <span>Kitty K. Wong</span>
  </div>
  <hr>
    
   
  <div class="post-content">
    <!--
    <div class="post-categories">
      <h2><span style="color:#1040a7">Paper Topic(s):</span></h2> 
      
      
      <a 
        href="kittywong/categories#Population Imaging and Imaging Genetics"
        class="post-category">
        Population Imaging and Imaging Genetics
      </a>
      
      <a 
        href="kittywong/categories#Integration of Imaging with Non-Imaging Biomarkers"
        class="post-category">
        Integration of Imaging with Non-Imaging Biomarkers
      </a>
      
    
    </div>
    <div class="post-tags">
      
      <h2><span style="color:#1040a7">Author(s):</span></h2> 
      
      <a href="kittywong/tags#Bonazzola, Rodrigo"
        class="post-tags">
        Bonazzola, Rodrigo
      </a> |  
      
      <a href="kittywong/tags#Ravikumar, Nishant"
        class="post-tags">
        Ravikumar, Nishant
      </a> |  
      
      <a href="kittywong/tags#Attar, Rahman"
        class="post-tags">
        Attar, Rahman
      </a> |  
      
      <a href="kittywong/tags#Ferrante, Enzo"
        class="post-tags">
        Ferrante, Enzo
      </a> |  
      
      <a href="kittywong/tags#Syeda-Mahmood, Tanveer"
        class="post-tags">
        Syeda-Mahmood, Tanveer
      </a> |  
      
      <a href="kittywong/tags#Frangi, Alejandro F."
        class="post-tags">
        Frangi, Alejandro F.
      </a> |  
      
    </div>
  -->
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style>

    .sidenav {
      width: 200px;
      position: fixed;
      z-index: 1;
      top: 150px;
      left: 50px;
      background: #eee;
      overflow-x: hidden;
      padding: 0px 0;
    }
  
    .sidenav a {
      padding: 6px 8px 6px 16px;
      text-decoration: none;
      font-size: 16px;
      color: #2196F3;
      color: #FF4500;
      display: block;
    }
  
    .sidenav a:hover {
      color: #064579;
    }
    </style>
  </head>

  <div class="sidenav">
    <a href="#author-id">Paper Info</a>
    <a href="#review-id">Reviews</a>
    <a href="#metareview-id">Meta-Review(s)</a>
    <a href="#authorFeedback-id">Author Feedback</a>
    <a href="">Back to top</a>
    <a href="https://kittywong.github.io">Back to List of papers</a>
  </div>

    <table>
  <tbody>
    <tr>
      <td><a href="#author-id"><span style="color:#ff9900"><b>Paper Info</b></span></a></td>
      <td><a href="#review-id"><span style="color:#ff9900"><b>Reviews</b></span></a></td>
      <td><a href="#metareview-id"><span style="color:#ff9900"><b>Meta-review(s)</b></span></a></td>
      <td><a href="#authorFeedback-id"><span style="color:#ff9900"><b>Author Feedback</b></span></a></td>
    </tr>
  </tbody>
</table>

<h1 id="author-id">Authors</h1>
<p>Rodrigo Bonazzola, Nishant Ravikumar, Rahman Attar, Enzo Ferrante, Tanveer Syeda-Mahmood, Alejandro F. Frangi
<br /><br /></p>

<h1 id="abstract-id">Abstract</h1>
<p>Prospective studies with linked image and genetic data, suchas the UK Biobank (UKB), provide an unprecedented opportunity to extract knowledge on the genetic basis of image-derived phenotypes. However, the extent of phenotypes tested within so-called genome-wide association studies (GWAS) is usually limited to handcrafted features, where the main limitation to proceed otherwise is the high dimensionality of both the imaging and genetic data. Here, we propose an approach where the phenotyping is performed in an unsupervised manner, via autoencoders that operate on image-derived 3D meshes. Therefore, the latent variables produced by the encoder condense the information related to the geometry of the biologic structure of interest. The network’s training proceeds in two steps: the first is genotype-agnostic and the second enforces an association with a set of genetic markers selected via GWAS on the intermediate latent representation. This genotype-dependent optimisation procedure allows the refinement of the phenotypes produced by the autoencoder to better understand the effect of the genetic markers encountered. We tested and validated our proposed method on left-ventricular meshes derived from cardiovascular magnetic resonance images from the UKB, leading to the discovery of novel genetic associations that, to the best of our knowledge, had not been yet reported in the literature on cardiac phenotypes.
<br /><br /></p>

<h1 id="link-id">Link to paper</h1>
<p><a href="https://doi.org/10.1007/978-3-030-87240-3_67">https://doi.org/10.1007/978-3-030-87240-3_67</a>
<br /><br /></p>

<h1 id="code-id">Link to the code repository</h1>
<p>N/A
<br /><br /></p>

<h1 id="dataset-id">Link to the dataset(s)</h1>
<p>N/A
<br /><br /></p>

<hr />
<h1 id="review-id">Reviews</h1>

<h3 id="review-1">Review #1</h3>

<ul>
  <li><strong>Please describe the contribution of the paper</strong>
    <blockquote>
      <p>This paper proposes an unsupervised deep learning method to study the genetic basis of CMR-derived phenotypes. The method contains two-step. The first step is to extract intermediate representations of imaging data, and the second step is to study the association between this intermediate variable and the genetic data.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting.</strong>
    <blockquote>
      <p>This paper proposed an unsupervised learning method to extract the respretations of the imaging data, and then used the phenotype data to conduct a GWAS to analyze the association between imaging and genetics.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work.</strong>
    <blockquote>
      <ol>
        <li>The motivation is not clearly presented. It looks hard to interpret the extracted imaging feature.</li>
        <li>There were many grammatical mistakes.</li>
      </ol>
    </blockquote>
  </li>
  <li><strong>Please rate the clarity and organization of this paper</strong>
    <blockquote>
      <p>Poor</p>
    </blockquote>
  </li>
  <li><strong>Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance</strong>
    <blockquote>
      <p>Fair.</p>
    </blockquote>
  </li>
  <li><strong>Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: <a href="https://miccai2021.org/en/REVIEWER-GUIDELINES.html">https://miccai2021.org/en/REVIEWER-GUIDELINES.html</a></strong>
    <blockquote>
      <p>Please clearly present the motivation of this work. Do you want to identify new risk loci? Or do you want to extract a good intermediate imaging features? As an deep learning method, how about its performance improvement compared to conventional GWAS?</p>
    </blockquote>
  </li>
  <li><strong>Please state your overall opinion of the paper</strong>
    <blockquote>
      <p>strong reject (2)</p>
    </blockquote>
  </li>
  <li><strong>Please justify your recommendation. What were the major factors that led you to your overall score for this paper?</strong>
    <blockquote>
      <p>Confident.</p>
    </blockquote>
  </li>
  <li><strong>What is the ranking of this paper in your review stack?</strong>
    <blockquote>
      <p>8</p>
    </blockquote>
  </li>
  <li><strong>Number of papers in your stack</strong>
    <blockquote>
      <p>3</p>
    </blockquote>
  </li>
  <li><strong>Reviewer confidence</strong>
    <blockquote>
      <p>Very confident</p>
    </blockquote>
  </li>
</ul>

<p><br /><br /></p>

<h3 id="review-2">Review #2</h3>

<ul>
  <li><strong>Please describe the contribution of the paper</strong>
    <blockquote>
      <p>The manuscript proposed an innovative autoencoder model based on graph convolutional neural networks and optimized the model using a collaborative approach.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting.</strong>
    <blockquote>
      <p>It is very innovative approach to integrate imaging and genetics using autoencoder with collaborative learning, as presented in Fig 1. The author well described the motivation and contibution of the proposed model.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work.</strong>
    <blockquote>
      <p>It would be better if the author presents the rich comparisons with several baseline algorithm, such as CCA or PCA.</p>
    </blockquote>
  </li>
  <li><strong>Please rate the clarity and organization of this paper</strong>
    <blockquote>
      <p>Excellent</p>
    </blockquote>
  </li>
  <li><strong>Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance</strong>
    <blockquote>
      <p>The paper clearly described their implementations.</p>
    </blockquote>
  </li>
  <li><strong>Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: <a href="https://miccai2021.org/en/REVIEWER-GUIDELINES.html">https://miccai2021.org/en/REVIEWER-GUIDELINES.html</a></strong>
    <blockquote>
      <p>The paper is overall nicely written with interesting ideas. However, there are several questions I had that could help me better assess the value of the proposed methods, especially in the experiment. There’s no need for running additional experiments for any of my questions.</p>
    </blockquote>
  </li>
  <li><strong>Please state your overall opinion of the paper</strong>
    <blockquote>
      <p>strong accept (9)</p>
    </blockquote>
  </li>
  <li><strong>Please justify your recommendation. What were the major factors that led you to your overall score for this paper?</strong>
    <blockquote>
      <p>The proposed model is very innovative. In the manuscript, the author proposed a novel cost function to integrate imaging and genetics. The motivation of the cost function was very concrete.</p>
    </blockquote>
  </li>
  <li><strong>What is the ranking of this paper in your review stack?</strong>
    <blockquote>
      <p>1</p>
    </blockquote>
  </li>
  <li><strong>Number of papers in your stack</strong>
    <blockquote>
      <p>3</p>
    </blockquote>
  </li>
  <li><strong>Reviewer confidence</strong>
    <blockquote>
      <p>Very confident</p>
    </blockquote>
  </li>
</ul>

<p><br /><br /></p>

<h3 id="review-3">Review #3</h3>

<ul>
  <li><strong>Please describe the contribution of the paper</strong>
    <blockquote>
      <p>The paper presents an auto-encoder of 3D surface mesh models of the heart’s left ventricle that is simultaneously optimized to improve the odds of a significant correlation in a Genome-Wide Association Study. The authors train the encoder in two stages: first, by using graph convolutions on the mesh based on the graph Laplacian, and second by adding a term expressing preference for encoding components that have greater correlation with a selected set of SNPs. The authors train, validate, and test the pipeline on 29000 subjects from the UK Biobank, claiming to discover a new heart-related SNP.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main strengths of the paper; you should write about a novel formulation, an original way to use data, demonstration of clinical feasibility, a novel application, a particularly strong evaluation, or anything else that is a strong aspect of this work. Please provide details, for instance, if a method is novel, explain what aspect is novel and why this is interesting.</strong>
    <blockquote>
      <p>The paper addresses an important area in imaging genetics, i.e. maximizing discoverability. GWAS is a bit old-hat, but remains a useful screening tool for yet-undiscovered variants, knowledge of which can be passed on to basic science. The choice of spectral embedding with a Chebyshev polynomial kernel is reasonable, and the pre-processing of the data appears to be done thoroughly and well. The learning &amp; GWAS study is done on a sufficiently large sample, and the validation of the method is solid, with varying seeds for splitting the data and assessing the p-value distribution for a each of a set of hyperparameters. That the authors connect their putative discovery with an interpretable trait (sphericity) by way of the auto-encoder component is also a bonus.</p>
    </blockquote>
  </li>
  <li><strong>Please list the main weaknesses of the paper. Please provide details, for instance, if you think a method is not novel, explain why and provide a reference to prior work.</strong>
    <blockquote>
      <p>The novelty of the method is somewhat limited, although this can be forgiven, given the goal of improving GWAS discovery. The main weakness of the method is that it relies on optimizing the encoder components’ correlation with SNP’s that are already sufficiently strongly correlated to pass Bonferroni correction. This type of approach appears circular in its reasoning. Although not circular in the statistical/ML sense, this is still a recipe for spurious findings. To properly optimize complex traits for “discoverability,” the process has to be agnostic to specific variants, especially from the very same dataset. More sensible approaches target general heritability, defined for example by GCTA https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363/. Even approaches like this https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5946904/ use overall GWAS from external data to define “more discoverable” traits. 
The most likely outcome here is that the same variant would be putatively discovered without the second (genetic) stage of the auto-encoder training.
As a more minor point, the autoencoder is comprised of only 4 components in the end (after stage 1), which makes the entire procedure seem somewhat unnecessary, given the low dimensionality to begin with. 
Also, it seems the initial auto-encoding was done on the entire dataset at once (although this is not totally clear from the algorithm outline), which would obviously bias the results further.</p>

    </blockquote>
  </li>
  <li><strong>Please rate the clarity and organization of this paper</strong>
    <blockquote>
      <p>Very Good</p>
    </blockquote>
  </li>
  <li><strong>Please comment on the reproducibility of the paper. Note, that authors have filled out a reproducibility checklist upon submission. Please be aware that authors are not required to meet all criteria on the checklist - for instance, providing code and data is a plus, but not a requirement for acceptance</strong>
    <blockquote>
      <p>Authors include code in the supplement &amp; describe the tools for pre-processing. Data is UKBB, so can be accessed with some effort by any researcher.</p>
    </blockquote>
  </li>
  <li><strong>Please provide detailed and constructive comments for the authors. Please also refer to our Reviewer’s guide on what makes a good review: <a href="https://miccai2021.org/en/REVIEWER-GUIDELINES.html">https://miccai2021.org/en/REVIEWER-GUIDELINES.html</a></strong>
    <blockquote>
      <p>See weaknesses</p>
    </blockquote>
  </li>
  <li><strong>Please state your overall opinion of the paper</strong>
    <blockquote>
      <p>borderline reject (5)</p>
    </blockquote>
  </li>
  <li><strong>Please justify your recommendation. What were the major factors that led you to your overall score for this paper?</strong>
    <blockquote>
      <p>At the end, the paper addresses an important issue, but the simplistic approach to the genomics aspect makes this a work of limited value.</p>
    </blockquote>
  </li>
  <li><strong>What is the ranking of this paper in your review stack?</strong>
    <blockquote>
      <p>3</p>
    </blockquote>
  </li>
  <li><strong>Number of papers in your stack</strong>
    <blockquote>
      <p>3</p>
    </blockquote>
  </li>
  <li><strong>Reviewer confidence</strong>
    <blockquote>
      <p>Very confident</p>
    </blockquote>
  </li>
</ul>

<p><br /><br /></p>

<hr />
<h1 id="metareview-id">Meta-Review(s)</h1>

<h2 id="primary-meta-review">Primary Meta-Review</h2>
<ul>
  <li><strong>Please provide your assessment of this work, taking into account all reviews. Summarize the key strengths and weaknesses of the paper and justify your recommendation. In case you deviate from the reviewers’ recommendations, explain in detail the reasons why. In case of an invitation for rebuttal, clarify which points are important to address in the rebuttal.</strong>
    <blockquote>
      <p>The reviewers recognize the interest of the presented approach to enhance discoverability of relevant imaging-genetics associations. The presented autoencoding framework was found sound while the experimental assessment on the UK Biobank is compelling. The paper seems however to suffer from lack of clarity in illustrating the benefits from the proposed two steps approach over the conventional GWAS analysis. There seems to be an issue with the circularity of the analysis, as both screening and validation of variants are performed on the same dataset.</p>
    </blockquote>
  </li>
  <li><strong>What is the ranking of this paper in your stack? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers).</strong>
    <blockquote>
      <p>5</p>
    </blockquote>
  </li>
  <li><strong>Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores,  indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision.</strong>
    <blockquote>
      <p>The relevance of the approach is discussed, in particular with respect to the optimization results shown in Figure 3. Concerning the circularity issue, it is made clear that the multi-step approach is performed on disjoint discovery and validation sets.</p>

      <p>Overall, the rebuttal is satisfactory in addressing the major concerns expressed during the review. Due to the originality of the proposed approach, and the large sample considered in the study, the feeling is that the work could make a nice contribution to the conference.</p>
    </blockquote>
  </li>
  <li><strong>After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal.</strong>
    <blockquote>
      <p>Accept</p>
    </blockquote>
  </li>
  <li><strong>What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers).</strong>
    <blockquote>
      <p>7</p>
    </blockquote>
  </li>
</ul>

<p><br /><br /></p>

<h2 id="meta-review-2">Meta-Review #2</h2>
<ul>
  <li><strong>Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores,  indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision.</strong>
    <blockquote>
      <p>The authors have addressed the clarity with respect to the key circularity issue.</p>
    </blockquote>
  </li>
  <li><strong>After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal.</strong>
    <blockquote>
      <p>Accept</p>
    </blockquote>
  </li>
  <li><strong>What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers).</strong>
    <blockquote>
      <p>6</p>
    </blockquote>
  </li>
</ul>

<p><br /><br /></p>
<h2 id="meta-review-3">Meta-Review #3</h2>
<ul>
  <li><strong>Please provide your assessment of the paper taking all information into account, including rebuttal. Highlight the key strengths and weaknesses of the paper, clarify how you reconciled contrasting review comments and scores,  indicate if concerns were successfully addressed in the rebuttal, and provide a clear justification of your decision. If you disagree with some of the (meta)reviewer statements, you can indicate so in your meta-review. Please make sure that the authors, program chairs, and the public can understand the reason for your decision.</strong>
    <blockquote>
      <p>This paper proposes to learn genetic associations with image-based phenotypes using unsupervised graph auto encoder. Validation is on left ventricular shapes from the uk biobank.</p>

      <p>One reviewer finds the motivation unclear.</p>

      <p>A second reviewer finds it clear but finds it lacking better comparisons.</p>

      <p>A third reviewer appreciates the method but questions whether discoveries may be due to circular reasoning.</p>

      <p>Reviews are extremely disagreeing, however, two of which are too short, and in my opinion, fail to meet expectations from a reviewer. The most constructive review brings a point on circularity. In other words, optimizing the encoder components with genetic snp’s may lead to spurious discoveries. From my understanding of the paper and from the author’s rebuttal, the approach is not to necessarily discover new snp’s but rather provided richer, fine-tuning of phenotyping shape descriptors. The algorithmic contribution uses existing methods, less original in that aspect, but their use to fine-tune phenotyping with an extensive validation is, in my opinion, a contribution with potential impact worth publishing.</p>

      <p>For these reasons, Recommendation is toward Acceptance.</p>
    </blockquote>
  </li>
  <li><strong>After you have reviewed the rebuttal, please provide your final rating based on all reviews and the authors’ rebuttal.</strong>
    <blockquote>
      <p>Accept</p>
    </blockquote>
  </li>
  <li><strong>What is the rank of this paper among all your rebuttal papers? Use a number between 1 (best paper in your stack) and n (worst paper in your stack of n papers).</strong>
    <blockquote>
      <p>11</p>
    </blockquote>
  </li>
</ul>

<h2><br /><br /></h2>
<h1 id="authorFeedback-id">Author Feedback</h1>
<blockquote>
  <p>We thank all the reviewers for their comments. Overall, they seem to concur on the importance of this emerging field and our general approach/novelty. The main comments to be addressed concern circularity and comparison to older statistical methods.
To clarify for Reviewer-1,   our method in  the training consists of 3 (not two as stated in the review) steps: the first step extracts intermediate representations of imaging data, the second step studies the association between the shape features and the genetic data using GWAS, the third step incorporates the salient SNP found back into the estimation of shape features through a new loss function. Finally, during inference, we show the improvement in the strength of the association of the shape features with the genomic loci from the updated shape features.</p>

  <p>Reviewer-2: Although it was not clear from the comment, we assume the comparison reference to PCA was for the shape representation and CCA was for the purpose of GWAS association. Our prior work using statistical methods such as PCA instead of the graph convolutional mesh revealed no meaningful GWAS associations. Since the mesh encoder captures the relationship between voxels inherent in the finite element mesh modeling, the resulting embedding is more suitable for finding the associations in GWAS.  Our focus was on discovering relevant shape features associated with genetic loci without pre-programming GWAS with selected features as done in conventional GWAS. However, it is possible that CCA will be useful  for GWAS itself although sparse sample graph-based CCA on such large dimensions will be computationally expensive requiring matrix deflation techniques.</p>

  <p>Reviewer-3: We emphasize that, to our knowledge, our paper is the first work to extract deep shape features for genetic discovery using unsupervised geometric deep learning. It makes two specific novel contributions: 1) use of variationally regularized convolutional mesh autoencoders on graphs for the purpose of discovery of genetic associations of shape features. 2) using a genome-dependent loss function to optimize the shape representation for genetic association.
Previous work on 3D facial meshes relied on more traditional ML approaches like hierarchical clustering and PCA on shape for GWAS association.</p>

  <p>On the circularity question, we clarify that the purpose of the second training step is to find a refined representation (i.e. a better phenotype) for association with genetics, which stresses the morphological changes that led to increase such association. It was not intended to discover newer gene loci. We found that a term enforcing correlation of a latent variable with SNP dosage is indeed effective in improving the association in an independent sample. This is not obvious a priori due to the small effect of single genetic variants on the phenotype. Finally, there is no circularity in the statistical/ML sense, since no optimization is performed on the GWAS sample.  We can assume a uniform distribution of p-values under the null distribution of no effect, which in turn allows us to perform hypothesis testing in the usual way without increasing the risk of spurious findings. While the improvement in association is modest, it is still statistically significant as shown in Fig. 3, with w_{SNP}=0 corresponding to no optimization; this supports its inclusion in the paper. We believe that by taking special measures during training, as well as using a larger sample size, the optimization can be further improved. We further argue that in a more general scenario, this refinement could also help find more true-positive associations for the case of a polygenic trait.
We also clarify that the testing of the improved association (i.e. in final GWAS) was performed on a disjoint patient dataset from the one used for initial encoding, thus avoiding the bias due to overlap of datasets. 
The paper will be thoroughly revised for any grammatical errors in the final version.</p>
</blockquote>

<p><br /><br />
<a href=""><span style="color:#ff9900"><b>back to top</b></span></a></p>

<hr />


  </div>

  <div class="post-info">
    <!--
    <div class="post-date">
      Poster presentation date:  
      0566-12-31
      -->
      <!--
      
        ,
        updated at 
        0567-01-01
      
      .
      
    </div>
    -->
    <!--
    
    <div class="post-author">
      Author: Kitty K. Wong
    </div>
    
    -->
    <div class="post-categories">
      <span><b>Topic(s): </b></span>
      
      <a 
        href="kittywong/categories#Population Imaging and Imaging Genetics"
        class="post-category">
        Population Imaging and Imaging Genetics
      </a> |
      
      <a 
        href="kittywong/categories#Integration of Imaging with Non-Imaging Biomarkers"
        class="post-category">
        Integration of Imaging with Non-Imaging Biomarkers
      </a> |
      
    </div>
    <div class="post-tags">
      <!--<span>Author List</span>-->
      <span><b> Author(s): </b></span>
      
      <a href="kittywong/tags#Bonazzola, Rodrigo"
        class="post-category">
        Bonazzola, Rodrigo
      </a> |  
      
      <a href="kittywong/tags#Ravikumar, Nishant"
        class="post-category">
        Ravikumar, Nishant
      </a> |  
      
      <a href="kittywong/tags#Attar, Rahman"
        class="post-category">
        Attar, Rahman
      </a> |  
      
      <a href="kittywong/tags#Ferrante, Enzo"
        class="post-category">
        Ferrante, Enzo
      </a> |  
      
      <a href="kittywong/tags#Syeda-Mahmood, Tanveer"
        class="post-category">
        Syeda-Mahmood, Tanveer
      </a> |  
      
      <a href="kittywong/tags#Frangi, Alejandro F."
        class="post-category">
        Frangi, Alejandro F.
      </a> |  
      
    </div>
    <div class="post-other">
      
      <div>
        <span><b>
          Next paper: 
        </b>
        </span>
        <a href="/0567/12/31/Paper1837">
          GKD: Semi-supervised Graph Knowledge Distillation for Graph-Independent Inference
        </a>
      </div>
      
      
      <div>
        <span><b>
          Previous paper: 
        </b>
        </span>
        <a href="/0565/12/31/Paper0680">
          Combining 3D Image and Tabular Data via the Dynamic Affine Feature Map Transform
        </a>
      </div>
      
    </div>
    
  </div>
</article>

      </main>
      <footer class="container-footer">
        <div class="footer-copyright">
  <span class="footer-copyright-text float-left">
    Copyright &copy; 2021. Kitty K. Wong, The MICCAI Society
  </span>
</div>
<!--
<div class="footer-copyright">
  <span class="footer-copyright-text float-left">
    Copyright &copy; 2021. example.com.
  </span>
  <span class="footer-copyright-text float-right">
    Powered by <a href="https://jekyllrb.com/">Jekyll</a>, themed by <a href="https://github.com/ghosind/Jekyll-Paper-Github">Jekyll-Paper-Github</a>.
  </span>
</div>
-->

      </footer>
    </div>
  </body>
</html>
